tests/format/js/pipeline-operator/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -257,34 +257,34 @@ function createPerson(attrs) {
 
 const result =
   [1, 2, 3]
-  |> %.map((a) => a * 2)
-  |> %.filter((a) => a > 5)
-  |> %.reduce((sum, a) => a + sum, 0)
-  |> increment(%)
-  |> add(%, 3);
+    |> %.map((a) => a * 2)
+    |> %.filter((a) => a > 5)
+    |> %.reduce((sum, a) => a + sum, 0)
+    |> increment(%)
+    |> add(%, 3);
 
 const searchResults$ =
   fromEvent(document.querySelector("input"), "input")
-  |> map(%, (event) => event.target.value)
-  |> filter(%, (searchText) => searchText.length > 2)
-  |> debounce(%, 300)
-  |> distinctUntilChanged(%)
-  |> switchMap(%, (searchText) => queryApi(searchText) |> retry(%, 3))
-  |> share(%);
+    |> map(%, (event) => event.target.value)
+    |> filter(%, (searchText) => searchText.length > 2)
+    |> debounce(%, 300)
+    |> distinctUntilChanged(%)
+    |> switchMap(%, (searchText) => queryApi(searchText) |> retry(%, 3))
+    |> share(%);
 
 v |> %.method() |> f(%);
 
 async function* f() {
   return (
     x
-    |> (yield %)
-    |> (await %)
-    |> y(%)
-    |> a.b(%)
-    |> a.b(%)
-    |> a.b(%)
-    |> a.b?.(%)
-    |> a.b?.(%)
+      |> (yield %)
+      |> (await %)
+      |> y(%)
+      |> a.b(%)
+      |> a.b(%)
+      |> a.b(%)
+      |> a.b?.(%)
+      |> a.b?.(%)
```

Comment:
@fisker I've updated snapshots with indentation version (https://github.com/prettier/prettier/pull/11557#discussion_r718039101). What do you  think this change?